lib/spaceship/tunes/tunes_client.rb

```diff
@@ -393,6 +393,36 @@ def price_tier(app_id)
       end
     end
 
+    # Returns an array of all available pricing tiers
+    # @return [Array] the PricingTier objects
+    # [{
+    #   "tierStem": "0",
+    #   "tierName": "Free",
+    #   "pricingInfo": [{
+    #       "country": "United States",
+    #       "countryCode": "US",
+    #       "currencySymbol": "$",
+    #       "currencyCode": "USD",
+    #       "wholesalePrice": 0.0,
+    #       "retailPrice": 0.0,
+    #       "fRetailPrice": "$0.00",
+    #       "fWholesalePrice": "$0.00"
+    #     }, {
+    #     ...
+    # }, {
+    # ...
+    def pricing_tiers
```

Comment:
@Aaang yes the redesign is something I took note in #219 and I want to tackle this in the coming weeks, if noone does it before me :)